### PR TITLE
create Rcpp template file + minor editting

### PR DESCRIPTION
**Pull request by kevinushey**

### CODE REVIEW

**Comment by jjallaire** [comment]
I think the package will fail R CMD check because there are no docs for rcpp_hello_world, correct?

File: src/cpp/session/modules/SessionPackages.R [hunk 1]
```diff
@@ -761,6 +761,53 @@ if (identical(as.character(Sys.info()["sysname"]), "Darwin") &&
...
+            "// c(0, 1).",
+            "",
+            "// The comment '// [[Rcpp::export' immediately above the function",
+            "// signals that this function should automatically be exported to",
+            "// R when the package is built.",
+            "//",
+            "// [[Rcpp::export]]",
+            "List rcpp_hello_world() {",
+            "",
+            "    CharacterVector x = CharacterVector::create(\"foo\", \"bar\");",
```
Comment (jjallaire) [inline src/cpp/session/modules/SessionPackages.R]: Let's indent by 2 spaces to match the RStudio default.

File: src/cpp/session/modules/SessionPackages.R [hunk 2]
```diff
@@ -761,6 +761,53 @@ if (identical(as.character(Sys.info()["sysname"]), "Darwin") &&
...
+            "",
+            "// This is a simple function using Rcpp that",
+            "// creates an R list containing a character vector",
+            "// c(\"foo\", \"bar\") and a numeric vector",
+            "// c(0, 1).",
+            "",
+            "// The comment '// [[Rcpp::export' immediately above the function",
+            "// signals that this function should automatically be exported to",
+            "// R when the package is built.",
+            "//",
```
Comment (jjallaire) [inline src/cpp/session/modules/SessionPackages.R]: Let's separate the explanatory comments from the Rcpp::export comment just for clarity

File: src/cpp/session/modules/SessionPackages.R [hunk 3]
```diff
@@ -844,6 +891,17 @@ if (identical(as.character(Sys.info()["sysname"]), "Darwin") &&
    # into using devtools won't need the template file)
    if (file.exists(file.path(packageDirectory, "R", "hello.R")))
       .Call("rs_addFirstRunDoc", RprojPath, "R/hello.R")
+
+   ## NOTE: This must come last to ensure the other package
+   ## infrastructure bits have been generated; otherwise
+   ## compileAttributes can fail
+   if (usingRcpp &&
+       require(Rcpp, quietly = TRUE) &&
```
Comment (jjallaire) [inline src/cpp/session/modules/SessionPackages.R]: For this we should just use:

```r
.rs.isPackageVersionInstalled("Rcpp", "0.10.1")
```

**Comment by kevinushey** [comment]
Yes, you are correct -- I added a sample `.Rd` file and the skeleton package now builds and checks fine (except with a warning for the `LICENSE` field, which we opted to live with)

**Pull request merged**